dashboard/src/modules/components/TableComponent/index.jsx

```diff
@@ -20,21 +15,20 @@ import {
   Tbody,
   Td,
 } from "@patternfly/react-table";
-import BarsIcon from "@patternfly/react-icons/dist/js/icons/bars-icon";
 import SearchBox from "../SearchComponent";
 import DatePickerWidget from "../DatePickerComponent";
 import Heading from "../HeadingComponent";
 import PathBreadCrumb from "../BreadCrumbComponent";
 import AlertMessage from "../AlertComponent";
-import NavItems from "../NavbarComponent";
 import EmptyTable from "../EmptyStateComponent";
 import moment from "moment";
 import { fetchPublicDatasets } from "../../../actions/fetchPublicDatasets";
 import TablePagination from "../PaginationComponent";
 import { constructUTCDate } from "../../../utils/constructDate";
 import { formatDate } from "../../../utils/dateFormatter";
-let startDate =constructUTCDate(new Date(1990, 10, 4));
-let endDate = constructUTCDate(new Date(formatDate(new Date())))
+import MainLayout from "../../containers/MainLayout";
+let startDate = new Date(Date.UTC(1990, 10, 4));
+let endDate = constructUTCDate(new Date(formatDate(new Date())));
```

Comment:
Line 31 looks a bit tortured.

Why wouldn't just this be adequate:
```
let endDate = new Date();
```
If you really need to adjust it to midnight, perhaps something like this would do:
```
const now = new Date();
let endDate = new Date(Date.UTC(now.getUTCFullYear(), now.getUTCMonth(), now.getUTCDate()));
```
although, you might want to add `1` to the third parameter value.

Reply:
This is another spot which would benefit by having `constructUTCDate()` use the current date/time as the default if no value is specified.